src/transformers/tokenization_utils_fast.py

```diff
@@ -98,24 +99,28 @@ def __init__(self, *args, **kwargs):
         tokenizer_object = kwargs.pop("tokenizer_object", None)
         slow_tokenizer = kwargs.pop("__slow_tokenizer", None)
         gguf_file = kwargs.pop("gguf_file", None)
+        tiktoken_file = kwargs.pop("tiktoken_file", None)
```

Comment:
this is really just a flag, the file path is not actually needed. This is most consistent, as it matched the gguf_file support, but maybe there is a less redundant way to pass this!